src/hostapi/asio/pa_asio.cpp

```diff
@@ -1304,6 +1304,17 @@ PaError PaAsio_Initialize( PaUtilHostApiRepresentation **hostApi, PaHostApiIndex
                 continue;
             }
 
+            // Blacklist dynamic loading of JackRouter ASIO driver by 64-bit applications.
```

Comment:
Please move any changes unrelated to the purpose of this PR to a separate PR. In particular, blacklisting a device is unrelated to adding support for ASIO messages.

Reply:
I've removed the JackRouter blacklisting rule.  The issues relating to it on their GitHub remain closed but two years on it's my understanding that bug has long since been fixed and few computers will have the old broken version installed.